fragments/training/training-fragment.md

```diff
@@ -1,6 +1,68 @@
 # Model Training Fragment
 
-**THIS SECTION IS STILL UNDER DEVELOPMENT.**
+This document describes the structure and content of a Model Training object.
 
 | Field Name          | Type                | Description                                                           |
 |---------------------|---------------------|-----------------------------------------------------------------------|
+| `created_at`        | string              | **Required.** Approximate date and time at which the model was trained. It is formatted according to [RFC 3339, section 5.6].  |
+| `environment`       | [Environment]       | **Required.** Description of the environment used to train the model. |
+| `data`              | [string]            | **Required.** A list of URLs, each of which points to a [STAC Collection] representing source imagery used to train the model. |
```

Comment:
It's unclear whether these URLs should point to a static STAC collection or a STAC collection hosted on an API

Reply:
I can add something to make this more clear, but it should be able to be either one. I think as long as the URL resolves to a STAC Collection that implements the ML AOI Extension as described in the section below then it should be sufficient.

There is another question of how to manage links that require authentication. As far as I know, STAC doesn't have a clear way of handling this, so we may need to define something ourselves. Maybe the auth problem would be best addressed in a separate issue?

Reply:
For authentication, can't the collection level catalog be open and the items require authentication? 
I don't think this is a major concern for now, but this is what @kbgg implemented to our API to make the collections public and discoverable (on the API).

Reply:
> can't the collection level catalog be open and the items require authentication?

That's true, but there is nothing in the STAC spec that dictates how to handle this generally. This works in the MLHub API because we are working in the context of a single API. Our Python client is specific to that API, so it just adds the configured API key to the requests. The problem is a bit more difficult if we have a case where the ML AOI Collection is in one Catalog/API (say within the NASA CMR system or a public catalog), but the input data and label Items are in a different Catalog/API (say MLHub). In that case there would be 2 potentially different authentication systems at play and a client would need to know the specifics of each one to navigate the links.

> I don't think this is a major concern for now

I agree, and I think it might be best addressed at the level of the STAC spec/extensions anyway. I'll open an issue to track this, but I think we can proceed for now.

Reply:
@kbgg I pushed a change in e8a94864a4f300ea66190de0ddcfec65b1b72a46 to clarify that Collections can be either static or part of an API. Let me know if this makes sense.